src/components/RecordTypesSelect/components/RecordItem.js

```diff
@@ -66,7 +66,7 @@ export const RecordItem = memo(({
       data-test-record-item
       tabIndex="0"
       role="button"
-      id={incomingRecord.type}
+      data-id={incomingRecord.type}
```

Comment:
Move it to the top (to other `data` attributes), please

Reply:
done